types/injectpromise/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for injectpromise 1.0
+// Project: https://github.com/sullof/injectpromise
+// Definitions by: naeemy <https://github.com/naeemy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace injectpromise;
+
```

Comment:
why do you think this is an UMD like module?
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
```suggestion
```